packages/fxa-payments-server/src/lib/apiClient.test.ts

```diff
@@ -628,6 +651,29 @@ describe('API requests', () => {
       });
       requestMock.done();
     });
+
+    it('POST {auth-server}/v1/oauth/subscriptions/active/new-paypal with coupon', async () => {
+      const paramsWithPromo = Object.assign(params, { promotionCode: 'TEST' });
+
+      const requestMock = nock(AUTH_BASE_URL)
+        .post(path, paramsWithPromo)
+        .reply(200, MOCK_PAYPAL_SUBSCRIPTION_RESULT);
+      expect(await apiCapturePaypalPayment(paramsWithPromo)).toEqual(
+        MOCK_PAYPAL_SUBSCRIPTION_RESULT
+      );
+
+      expect(
+        <jest.Mock>createSubscriptionWithPaymentMethod_PENDING
+      ).toBeCalledWith(metricsOptions);
+      expect(
+        <jest.Mock>createSubscriptionWithPaymentMethod_FULFILLED
+      ).toBeCalledWith({
+        ...metricsOptions,
+        sourceCountry: 'FR',
+      });
```

Comment:
Both of these should also have a 'promotionCode' event property.

Reply:
updated